public/docs/ts/_cache/tutorial/toh-pt6.jade

```diff
@@ -183,173 +209,141 @@ block get-heroes-details
 
 - var rejected_promise = _docsFor == 'dart' ? 'propagated exception' : 'rejected promise';
 :marked
-  In this demo service we log the error to the console; we should do better in real life.
+  In this demo service we log the error to the console; we would do better in real life.
 
   We've also decided to return a user friendly form of the error to
   the caller in a !{rejected_promise} so that the caller can display a proper error message to the user.
 
-  ### !{_Promise}s are !{_Promise}s
+  ### Unchanged `getHeroes` API
+
   Although we made significant *internal* changes to `getHeroes()`, the public signature did not change.
   We still return a !{_Promise}. We won't have to update any of the components that call `getHeroes()`.
 
-.l-main-section
-:marked
-  ## Add, Edit, Delete
+  ### Add, edit, delete heroes
 
   Our stakeholders are incredibly pleased with the added flexibility from the API integration, but it doesn't stop there. Next we want to add the capability to add, edit and delete heroes.
 
-  We'll complete `HeroService` by creating `post`, `put` and `delete` methods to meet our new requirements.
+  We'll start with the edit feature since we have most of the capabilities already.
 
+.l-main-section#edit
 :marked
-  ### Post
+  ## Persist name changes to the server
```

Comment:
I was surprised this was `##`, not `###`. Also, this section title/content wasn't clearly telegraphed by what came before.